test/features/transaction/transaction_details_page_test.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:didpay/features/transaction/transaction.dart';
+import 'package:didpay/features/transaction/transaction_details_page.dart';
+import 'package:didpay/features/transaction/transaction_notifier.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:intl/intl.dart';
+
+import '../../helpers/mocks.dart';
+import '../../helpers/test_data.dart';
+import '../../helpers/widget_helpers.dart';
+import 'transaction_tile_test.dart';
```

Comment:
Instead of importing another test file, you could put the shared data in the `test_data.dart` file and import that in this test.

Reply:
+1!